### PR TITLE
Update overview xaml to present recent transactions.

### LINKED ISSUE #77: Recent transactions are not shown in the overview

### PR DESCRIPTION
**Pull request by jrick**
The recent txs were already being added and updated in the viewmodels.

There's a lot more that can be done here, such as expanding selected
transaction items to show more details.

Originally there were plans that selecting an item would replace the
overview with an entirely new view devoted to showing all of the
transaction's details.  Right now I think this disrupts the user
(especially since this is the only place in the app that we'd need
some kind of "back button").  Removing this view for now
(OverviewDeeper.xaml) and this can be figured out later.  What's more
important now is to put some moderately useful data here instead of
having a giant overview page with nothing but 2 numbers at the top.

Fixes #77.

### CODE REVIEW

**Comment by cjepson** [comment]
tACK, looks pretty.

File: Paymetheus/Overview.xaml [hunk 1]
```diff
@@ -1,143 +1,108 @@
...
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="60"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="1*"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition/>
+        </Grid.ColumnDefinitions>
+        <TextBlock Grid.Row="0" HorizontalAlignment="Left" Margin="0 0 0 20" Text="Totals" VerticalAlignment="Top" Foreground="#FF0C1E3E" FontSize="20"/>
```
Comment (cjepson) [inline Paymetheus/Overview.xaml:62]: At some point we need to start defining these styles in App.xaml as well, so they can be easily accessed and modified between views.

Comment (jrick) [inline Paymetheus/Overview.xaml:62]: while i agree this wasn't actually changed as part of the pr.  it just looks that way because lol xml diffs.

**Comment by marcopeereboom** [comment]
I do not like this at all.  Bunch of clutter without any additional value imo.  This information is available in history with greater detail.  Let's have a debate about this.

**Comment by marcopeereboom** [comment]
Despite my dislike of the current transaction display I do think this is a step forward.

tOK

**Pull request merged**